src/cmd_handler.cc

```diff
@@ -233,12 +233,16 @@ std::string CmdHandler::hDumpKv(DBWrap* target_dbw,
         return ss.str();
     }
 
+    static const size_t iterate_number_limit = 100;
+
     // Options.
     bool hex_key = false;
     bool prefix_match = false;
     bool iterate = false;
     size_t iterate_number = 1;
-    for (size_t ii=2; ii<tokens.size(); ++ii) {
```

Comment:
if write 
```
getrecord -s 1
```
 into jungle_cmd, `ii` should start from 1. it is not correct, pls let me know.